tests/licensedcode/test_plugin_license_detection.py

```diff
@@ -130,6 +130,24 @@ def test_license_match_extra_words_3_seq():
     check_json_scan(test_loc, result_file, regen=REGEN_TEST_FIXTURES)
 
 
+def test_license_match_extra_words_3_seq_without_copyrights():
```

Comment:
```suggestion
def test_license_match_extra_words_3_seq_with_copyrights():
```
The new test added for the new case should be the one which does not have perfect score yet. The olds tests that we added in https://github.com/aboutcode-org/scancode-toolkit/pull/4402 should remain as-is (only change in the expectation, with score and detection log improvements). The new case should be added in a new test.